2024.11: Beta release notes

frenck opened the pull request:
<!--
  You are amazing! Thanks for contributing to our project!
  Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).

  Before submitting your pull request, please verify that you have chosen the correct target branch,
  and that the PR preview looks fine and does not include unrelated changes.
-->
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->


ℹ️ Work in progress for the 2024.11 beta releases notes

## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards


<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

## Release Notes for Home Assistant Version 2024.11

- **New Features**
	- Introduced a drag-and-drop interface for creating and converting dashboards.
	- Added low-latency camera streaming using WebRTC.
	- New integrations added, including support for various devices like BleBox, LG ThinQ, and more.
	- Enhanced functionality for existing integrations, including new platforms and entities such as diagnostics and calendar features.

- **Documentation Updates**
	- Improved clarity and detail in integration documentation for various devices.
	- Added sections for new platform capabilities, such as diagnostics and calendar features.
	- Updated user guidance for setup and troubleshooting across multiple integrations, including detailed instructions for removing device services.

- **Bug Fixes**
	- Resolved issues related to event handling and integration functionalities.

- **Chores**
	- Updated code ownership for various integrations to reflect new contributors.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

frenck committed:
2024.11: Beta release notes

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11615 to 0.5.11618 (#35430)

Bumps [sorbet-runtime](https://github.com/sorbet/sorbet) from 0.5.11615 to 0.5.11618.
- [Release notes](https://github.com/sorbet/sorbet/releases)
- [Commits](https://github.com/sorbet/sorbet/commits)

---
updated-dependencies:
- dependency-name: sorbet-runtime
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump json from 2.7.3 to 2.7.4 (#35431)

Bumps [json](https://github.com/ruby/json) from 2.7.3 to 2.7.4.
- [Release notes](https://github.com/ruby/json/releases)
- [Changelog](https://github.com/ruby/json/blob/master/CHANGES.md)
- [Commits](https://github.com/ruby/json/compare/v2.7.3...v2.7.4)

---
updated-dependencies:
- dependency-name: json
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

arturpragacz committed:
Add config flow to Onkyo (#35399)

* Add config flow to Onkyo

* Fix formatting

Galorhallen committed:
Update govee light local docs (#35400)

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

alistair23 committed:
itegration: husqvarna_automower_ble: Initial commit (#30921)

Signed-off-by: Alistair Francis <alistair@alistair23.me>

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11618 to 0.5.11620 (#35446)

Bumps [sorbet-runtime](https://github.com/sorbet/sorbet) from 0.5.11618 to 0.5.11620.
- [Release notes](https://github.com/sorbet/sorbet/releases)
- [Commits](https://github.com/sorbet/sorbet/commits)

---
updated-dependencies:
- dependency-name: sorbet-runtime
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

moritz-john committed:
Fix typo in pushover.markdown (#35437)

* Fix typo in pushover.markdown

* Update pushover.markdown

karwosts committed:
Describe confirmation options for button entities rows (#35436)

* Update entities.markdown

* tiny tweak

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

unfug-at-github committed:
Changed behavior of statistics introduced with #129211 (#35410)

* Update statistics.markdown

Describe changed behavior regarding state report events introduced with #129211

* Update statistics.markdown

updated according to review comments

c0ffeeca7 committed:
Integration instance remove (#35449)

* Add steps on removing an integration instance

* Revert changes

karwosts committed:
Update map.markdown

MindFreeze committed:
typo

MindFreeze committed:
Update map.markdown for geolocation focus option (#35454)

mback2k commented:
I am not sure if it is okay to mention additional items, please tell me if I shouldn't do this in the future:

Breaking change due to a bugfix: [Fix visualization by inverting open/closed state of patio awnings](https://github.com/home-assistant/core/pull/128079)
Potential noteworthy feature addition: [Add light support to WMS WebControl pro](https://github.com/home-assistant/core/pull/128308)

klaasnicolaas commented on source/_posts/2024-11-06-release-202411.markdown:
```diff
@@ -0,0 +1,318 @@
+now marked as stable! 
+
+This means that, as of this release, when you create a new dashboard, you
+will get our awesome drop and drop experience!
```
```suggestion
will get our awesome drag and drop experience!
```

mrdarrengriffin committed:
Add GitHub's top public project by contributors in 2024 badge (#35458)

* Add GitHub's "Top open source project by contributors in 2024" badge 🎉

* Added fragment identifier

Commit:
Add Vacuum platform to Matter docs (#35456)

tr4nt0r committed:
Add calendars to Habitica integration (#35206)

zweckj committed:
Add smart standby documentation for lamarzocco (#35439)

* Add smart standby documentation for lamarzocco

* Update lamarzocco.markdown

alengwenus committed:
Replace lock sensors by switches (#35103)

thecode committed:
Add Switcher Mini model to Switcher supported devices (#35443)

mrdarrengriffin committed:
Fix heading anchor scroll and wrapping (#35453)

* Fix heading anchor scroll and wrapping

* Increase scroll-margin-top to allow for more room after scroll

* Further adjustment

* Reduced again to prioritise 1 or 2 line heading whitespace on scroll

swistakm committed:
blebox: add integration documentation for luxSensor (#35455)

bramkragten committed:
Merge branch 'current' into merge-current-in-next

c0ffeeca7 committed:
Remove integration: add missing screenshot (#35473)

dependabot[bot] committed:
build(deps-dev): bump sass-embedded from 1.80.4 to 1.80.5 (#35470)

Bumps [sass-embedded](https://github.com/sass-contrib/sass-embedded-host-ruby) from 1.80.4 to 1.80.5.
- [Commits](https://github.com/sass-contrib/sass-embedded-host-ruby/compare/v1.80.4...v1.80.5)

---
updated-dependencies:
- dependency-name: sass-embedded
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

bramkragten committed:
Merge current into next (#35477)

dependabot[bot] committed:
build(deps-dev): bump rubocop-ast from 1.32.3 to 1.33.0 (#35471)

Bumps [rubocop-ast](https://github.com/rubocop/rubocop-ast) from 1.32.3 to 1.33.0.
- [Release notes](https://github.com/rubocop/rubocop-ast/releases)
- [Changelog](https://github.com/rubocop/rubocop-ast/blob/master/CHANGELOG.md)
- [Commits](https://github.com/rubocop/rubocop-ast/compare/v1.32.3...v1.33.0)

---
updated-dependencies:
- dependency-name: rubocop-ast
  dependency-type: indirect
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11620 to 0.5.11625 (#35472)

Bumps [sorbet-runtime](https://github.com/sorbet/sorbet) from 0.5.11620 to 0.5.11625.
- [Release notes](https://github.com/sorbet/sorbet/releases)
- [Commits](https://github.com/sorbet/sorbet/commits)

---
updated-dependencies:
- dependency-name: sorbet-runtime
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

joostlek committed:
Fix release (#35480)

LG-ThinQ-Integration committed:
Add LG ThinQ integration (#35427)

* Add LG ThinQ integration

* Update source/_integrations/lg_thinq.markdown

Co-authored-by: coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>

* Update source/_integrations/lg_thinq.markdown

Co-authored-by: coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>

* Update lg_thinq.markdown

* Update source/_integrations/lg_thinq.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update lg_thinq.markdown

* Update lg_thinq.markdown

* Update lg_thinq.markdown

* Update lg_thinq.markdown

Correct overall supported properties for all devices.

---------

Co-authored-by: jangwon.lee <jangwon.lee@lge.com>
Co-authored-by: coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>
Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

bramkragten commented:
> I am not sure if it is okay to mention additional items, please tell me if I shouldn't do this in the future:
> 
> Breaking change due to a bugfix: [Fix visualization by inverting open/closed state of patio awnings](https://github.com/home-assistant/core/pull/128079) Potential noteworthy feature addition: [Add light support to WMS WebControl pro](https://github.com/home-assistant/core/pull/128308)

Thanks! If they are labeled with the correct labels, they will pop up :-)

joostlek committed:
Pin sass-embedded to 1.80.4 (#35485)

joostlek committed:
Fix release (#35486)

c0ffeeca7 committed:
Assist sentences: restructure (#35457)

* Assist sentences: restructure

* Apply suggestions from code review

Co-authored-by: Michael Hansen <hansen.mike@gmail.com>

---------

Co-authored-by: Michael Hansen <hansen.mike@gmail.com>

mrdarrengriffin committed:
Further improvements for fragment scrolling (#35481)

* Further fixes for fragment scrolling

* revert Gemfile.lock

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

joostlek committed:
Add sensors for Spotify (#35292)

* Add sensors for Spotify

* Update source/_integrations/spotify.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/_integrations/spotify.markdown

* Update source/_integrations/spotify.markdown

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

noahhusby committed:
Added audio output to Cambridge Audio entities (#35444)

miterion committed:
Add Lock Support to SwitchBot Cloud Integration (#32196)

* Add Switchbot Lock to Cloud Integration

* tweak to kick the CI

* Update source/_integrations/switchbot_cloud.markdown

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>
Co-authored-by: Joost Lekkerkerker <joostlek@outlook.com>

starkillerOG committed:
Reolink TCP push docs (#35483)

* Reolink TCP push docs

* tweak to kick the CI

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

agoode committed:
prometheus: Document new behavior of unavailable or unknown entities (#34632)

* prometheus: Document new behavior of unavailable or unknown entities

* removing docs referring to behaviour in previous versions of HA

As the documentation in general is focused on the current version.

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

OzGav committed:
Create music_assistant.markdown (#35487)

* Create music_assistant.markdown

* Update source/_integrations/music_assistant.markdown

Co-authored-by: Joost Lekkerkerker <joostlek@outlook.com>

* Update source/_integrations/music_assistant.markdown

Co-authored-by: Joost Lekkerkerker <joostlek@outlook.com>

* Remove actions for now

* Fix CI

* tweaks

* Update source/_integrations/music_assistant.markdown

---------

Co-authored-by: Joost Lekkerkerker <joostlek@outlook.com>
Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

bramkragten committed:
Merge branch 'current' into 2024.11

bramkragten committed:
Merge branch 'next' into 2024.11

bramkragten committed:
update notes

bramkragten committed:
Update 2024-11-06-release-202411.markdown

mback2k commented:
> > I am not sure if it is okay to mention additional items, please tell me if I shouldn't do this in the future:
> 
> > 
> 
> > Breaking change due to a bugfix: [Fix visualization by inverting open/closed state of patio awnings](https://github.com/home-assistant/core/pull/128079) Potential noteworthy feature addition: [Add light support to WMS WebControl pro](https://github.com/home-assistant/core/pull/128308)
> 
> 
> 
> Thanks! If they are labeled with the correct labels, they will pop up :-)

You are welcome! Doesn't seem like it with your latest changes. I guess it depends on how the processed PR marker is interpreted, based on merge or creation/numeric order.
